great_expectations/checkpoint/util.py

```diff
@@ -209,7 +209,7 @@ def get_runtime_batch_request(
     substituted_runtime_config: CheckpointConfig,
     validation_batch_request: Optional[dict] = None,
     ge_cloud_mode: bool = False,
-) -> Union[BatchRequest, RuntimeBatchRequest]:
+) -> Optional[Union[BatchRequest, RuntimeBatchRequest]]:
```

Comment:
Isn't it true that this can just read `Optional[BatchRequest]` because `RuntimeBatchRequest` is a `BatchRequest`?

Reply:
Regardless, I think this is more clear about what exactly is being returned. I just know that in argument type hints we have been only using `BatchRequest`. Should be we treating our input argument and return argument type hints differently?

Reply:
@NathanFarmer Nice catch -- updated.  Thanks!